src/main/scala/org/broadinstitute/dsde/agora/server/ga4gh/ModelSupport.scala

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.dsde.agora.server.ga4gh
+
+import org.broadinstitute.dsde.agora.server.AgoraConfig
+import org.broadinstitute.dsde.agora.server.ga4gh.Models.{Metadata, Tool, ToolClass, ToolDescriptor, ToolDescriptorType, ToolId, ToolVersion}
+import org.broadinstitute.dsde.agora.server.model.{AgoraEntity, AgoraEntityType, MethodDefinition}
+import wdl4s.WdlNamespaceWithWorkflow
+
+import scala.util.Success
+
+/**
+  * Support class to handle apply methods with some degree of business logic
+  */
+object ModelSupport {
+
+  final val organization = "The Broad Institute of Harvard and MIT"
+  final val verifiedSource = ""
+  final val version = "1.0.0"
+  final val apiVersion = version
```

Comment:
while I don't anticipate changing these anytime soon, these versions should not be tied together. We could update our version before the spec is updated.

Reply:
👍